sources/meta-ornl/recipes-support/dotnet-core/dotnet-core_5.0.x.inc

```diff
@@ -31,15 +31,10 @@ do_install() {
 # Only needed for running locally.
 do_install_append() {
     install -d ${D}/home/root/
-    cat >> ${D}/home/root/.bash_profile <<EOF
+    cat >> ${D}/etc/environment <<EOF
```

Comment:
Produces this:
```
NOTE: Executing Tasks
WARNING: vsdbg-16.8.11005.1-r0 do_populate_lic: vsdbg: No generic license file exists for: Microsoft_Software_License in any provider
ERROR: dotnet-core-5.0.301-r0 do_install: Execution of '/ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332' failed with exit code 2:
/ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332: 114: /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332: cannot create /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/image/etc/environment: Directory nonexistent
WARNING: exit code 2 from a shell command.

ERROR: Logfile of failure stored in: /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/log.do_install.16332
Log data follows:
| DEBUG: Executing python function extend_recipe_sysroot
| NOTE: Direct dependencies are ['/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-devtools/quilt/quilt-native_0.66.bb:do_populate_sysroot', '/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-core/zlib/zlib_1.2.11.bb:do_populate_sysroot', '/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-devtools/gcc/gcc-cross_9.3.bb:do_populate_sysroot', '/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-core/glibc/glibc_2.31.bb:do_populate_sysroot', '/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-devtools/gcc/gcc-runtime_9.3.bb:do_populate_sysroot', 'virtual:native:/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-devtools/pseudo/pseudo_git.bb:do_populate_sysroot', 'virtual:native:/ephemeral/yocto-ornl-dunfell/sources/poky/meta/recipes-devtools/patch/patch_2.7.6.bb:do_populate_sysroot']
| NOTE: Installed into sysroot: []
| NOTE: Skipping as already exists in sysroot: ['quilt-native', 'zlib', 'gcc-cross-arm', 'glibc', 'gcc-runtime', 'pseudo-native', 'patch-native', 'binutils-cross-arm', 'gmp-native', 'zlib-native', 'flex-native', 'autoconf-native', 'xz-native', 'libtool-native', 'linux-libc-headers', 'libmpc-native', 'texinfo-dummy-native', 'automake-native', 'mpfr-native', 'gnu-config-native', 'libgcc', 'attr-native', 'm4-native', 'gettext-minimal-native']
| DEBUG: Python function extend_recipe_sysroot finished
| DEBUG: Executing shell function do_install
| /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332: 114: /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332: cannot create /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/image/etc/environment: Directory nonexistent
| WARNING: exit code 2 from a shell command.
| ERROR: Execution of '/ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332' failed with exit code 2:
| /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332: 114: /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/temp/run.do_install.16332: cannot create /ephemeral/yocto-ornl-dunfell/build_ornl/tmp/work/cortexa9t2hf-neon-fslc-linux-gnueabi/dotnet-core/5.0.301-r0/image/etc/environment: Directory nonexistent
| WARNING: exit code 2 from a shell command.
| 
ERROR: Task (/ephemeral/yocto-ornl-dunfell/ornl-layers/meta-ornl/recipes-support/dotnet-core/dotnet-core_5.0.301.bb:do_install) failed with exit code '1'
WARNING: linux-variscite-5.4.85+gitAUTOINC+343970224e-r0 do_kernel_metadata: [NOTE]: defconfig was supplied both via KBUILD_DEFCONFIG and SRC_URI. Dropping SRC_URI defconfig
NOTE: Tasks Summary: Attempted 6799 tasks of which 6700 didn't need to be rerun and 1 failed.
NOTE: Writing buildhistory
NOTE: Writing buildhistory took: 4 seconds

Summary: 1 task failed:
  /ephemeral/yocto-ornl-dunfell/ornl-layers/meta-ornl/recipes-support/dotnet-core/dotnet-core_5.0.301.bb:do_install
Summary: There were 2 WARNING messages shown.
Summary: There was 1 ERROR message shown, returning a non-zero exit code.
```

Reply:
suggest not creating new folders in etc if you can avoid it.  I get the desire to do this:
https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix

I think it was just a ham-fisted edit because the install line above it needed to be updated as well...